nuxeo-features/nuxeo-automation/nuxeo-automation-io/src/main/java/org/nuxeo/ecm/automation/jaxrs/io/documents/JsonESDocumentWriter.java

```diff
@@ -245,18 +246,17 @@ protected static void writeProperties(JsonGenerator jg, DocumentModel doc, Strin
         if (prefix == null || prefix.length() == 0) {
             prefix = schema;
         }
-        prefix = prefix + ":";
+        JSONPropertyWriter currentWriter = writer.prefix(prefix);
```

Comment:
According to comment above, I keep the fluent methods to chain calls, but they don't return a new instance anymore. Si that I create a new `JSONPropertyWriter` here.